.changeset/angry-papayas-applaud.md

```diff
@@ -0,0 +1,5 @@
+---
+'@faustwp/core': patch
+---
+
+Bug Fix: useLogin access token `code` is now properly escaped.
```

Comment:
Could we add more context here? Also, this is for any access token, not just `useLogin`.